Add documentation to activate HashiCorp Flex (#399)
hashicorp/terraform-docs-common: website/docs/cloud-docs/overview/activate-flex.mdx

~> **Note**: All New HashiCorp organizations are automatically allocated $50 in trial credits. In the event the HashiCorp Flex account is not yet allocated, you can use the trial credits to temporarily activate **Standard Edition** for immediate access to TFC paid functionality like [Team Management](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/teams) or more concurrency.

![A view of the Terraform Cloud account summary showing billing summary and available balance](/img/docs/tfc-account-summary.png)

## Step 4: Edit the Terraform Cloud Organization Plan

Navigate to the TFC organization setting's **Plan & Billing** page. For TFC user accounts that are not unified with HCP user accounts, you will be asked to login to an HCP account that has the `admin` role in the HCP organization with HashiCorp Flex allocated.  This login is required to allow you to edit the TFC organization's plan.  

~> **Note**: If you have a unified HCP account, you do not need to login and TFC will automatically identify qualified HCP organizations that your unified HCP user account is a member of.

> Click **Edit Plan** to be brought to the Plan Activation page.  

Reviewer: ```suggestion
Click **Edit Plan** to go to the Plan Activation page.  
```

> If your HashiCorp user account has the `owner` or `admin` role in multiple HCP organizations, you will be asked to select which Billing Account to configure against.

Reviewer: ```suggestion
If your HashiCorp user account has the `owner` or `admin` role in multiple HCP organizations, you will need to select a billing account.
```

![A view prompting the user to choose the billing account to link against](/img/docs/tfc-plan-org-select.png)

After selecting a plan, confirm your selection and review the **Cost Estimate**.

> If you meet all the necessary [prerequisites](#prerequisites), the **Plan Selection** page will show “HashiCorp Flex” at the bottom left. If you see “Pay-as-you-go,” please review steps 1-3 or [contact HashiCorp support](https://support.hashicorp.com/hc/en-us/requests/new) for more assistance.

Reviewer: bold UI elements instead of quotes
